File: king-admin-java/src/main/java/com/oukingtim/service/TbDictService.java
package com.oukingtim.service;

import com.baomidou.mybatisplus.service.IService;
import com.oukingtim.domain.TbDict;

import java.util.List;


/**
 * Created by oukingtim
 */
public interface TbDictService extends IService<TbDict> {

    /**
     * 根据分类编号查询字典
     * @param classCode
     * @return
     */
    List<TbDict> findByClassCode(String classCode);
}

File: king-admin-kotlin/src/main/kotlin/com/oukingtim/service/TbDictService.kt
package com.oukingtim.service

import com.baomidou.mybatisplus.service.IService
import com.oukingtim.domain.TbDict

/**
 * Created by oukingtim
 */
interface TbDictService : IService<TbDict> {

    /**
     * 根据分类编号查询字典
     * @param classCode
     * *
     * @return
     */
    fun findByClassCode(classCode: String): List<TbDict>
}